src/lib/format/string.ts

```diff
@@ -27,7 +27,7 @@ export const field2Caption = (field: string): string =>
  * @param field
  */
 export const field2Title = (field: string): string =>
-  ucFirst(addSpaces(unHyphen(field)).toLowerCase());
```

Comment:
I'm not sure if this is the correct place to address this, if so: please note the following:

- Code relying on the current behavior may break.
- By removing the `unHyphen`, the return value (as described in the JSDoc) is now longer correct, the return value now will bee "Field_name" for "field_name".

I think that either:

- https://github.com/maykinmedia/open-archiefbeheer/issues/284 can be fixed within OAB itself.
- `Field2title` should be able to receive an `options` prop allowing to indicate the steps to process, e.g:

```
export const field2title = (field: string, options = {
  addSpaces: true,
  lowerCase: true,
  ucFirst: true,
  unHyphen: true
}): string => {
    // IMPLEMENTATION
    return ""
}
```

If the latter options is chosen, this might be a good opportunity to remove `field2caption` since it's almost identical to `field2title`. And with a refactor of the latter the same results should be able to be achieved.